<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>

 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>480</width><height>350</height></rect>
  </property>
  <property name="windowTitle">
   <string>Login Options</string>
  </property>

  <widget class="QWidget" name="centralwidget">



   <layout class="QVBoxLayout" name="mainLayout">

    <!-- TITLE -->
    <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text"><string>Select Login</string></property>
       <property name="alignment"><set>Qt::AlignCenter</set></property>
      </widget>
    </item>

    <!-- SPACER -->
    <item>
      <spacer name="spacer1">
       <property name="orientation"><enum>Qt::Vertical</enum></property>
       <property name="sizeHint" stdset="0">
         <size><width>20</width><height>15</height></size>
       </property>
      </spacer>
    </item>

    <!-- BUTTONS -->
    <item>
      <widget class="QPushButton" name="btnUserLogin">
       <property name="text"><string>User Login</string></property>
       <property name="minimumHeight"><number>38</number></property>
      </widget>
    </item>

    <item>
      <widget class="QPushButton" name="btnAdminLogin">
       <property name="text"><string>Admin Login</string></property>
       <property name="minimumHeight"><number>38</number></property>
      </widget>
    </item>

    <item>
      <widget class="QPushButton" name="btnBack">
       <property name="text"><string>Back</string></property>
       <property name="minimumHeight"><number>38</number></property>
      </widget>
    </item>

    <!-- SPACER -->
    <item>
      <spacer name="spacerBottom">
       <property name="orientation"><enum>Qt::Vertical</enum></property>
       <property name="sizeHint" stdset="0">
         <size><width>20</width><height>25</height></size>
       </property>
      </spacer>
    </item>

   </layout>

  </widget>
 </widget>

 <resources/>
 <connections/>
</ui>
